test/main.js

```diff
@@ -0,0 +1,86 @@
+import test from "ava";
+
+var i18nPlugin,
+    File,
+    fs,
+    check;
+
+test.before(t => {
+    i18nPlugin = require("../");
+    File = require("vinyl");
+    fs = require("fs");
+
+    check = function (file, stream, callback) {
+        stream.on("data", function (newFile) {
+            callback(newFile);
+        });
+
+        stream.write(file);
+        stream.end();
+    };
+});
+
+test("plugin is defined", t => {
+    t.truthy(i18nPlugin);
+});
+
+test("reporter is defined", t => {
+    t.truthy(i18nPlugin.reporter);
+});
+
+test("shouldn't return error structure", t => {
+    var file = new File({
+            path: "test/fixtures/empty.ejs",
+            contents: fs.readFileSync("test/fixtures/empty.ejs")
+        }),
+        stream = i18nPlugin();
+
+    check(file, stream, function (newFile) {
+        t.falsy(newFile.i18nlint);
+    });
+});
+
+test("should return file and error", t => {
+    var file = new File({
+            path: "test/fixtures/test.ejs",
+            contents: fs.readFileSync("test/fixtures/test.ejs")
+        }),
+        stream = i18nPlugin(),
+        result;
+
+    check(file, stream, function (newFile) {
+        result = newFile.i18nlint[0];
+        t.truthy(result.file);
+        t.truthy(result.error);
+    });
+});
+
+test("should return error structure", t => {

```

Comment:
jeśli chcesz w `ava` obsłużyć test asynchroniczny, to musisz użyć zapisu      `test.cb(...` lub ... `test('it works', async t => {` z await w środku testu